packages/shorebird_cli/lib/src/extensions/version.dart

```diff
@@ -0,0 +1,28 @@
+import 'package:pub_semver/pub_semver.dart';
+
+extension VersionParsing on Version {
```

Comment:
Minor but making this an extension on Version feels weird because we're introducing a static method so you don't get the benefit of the API looking like `Version.tryParse` and the implementation doesn't benefit from being an extension on version. It might be simpler and easiest to work with as a top-level function `Version? tryParseVersion(String value, {bool strict = true}) {...}`

Not a huge deal just leaving my thoughts.

Reply:
I like that better than what I have, updating now